### ISSUE TITLE
[homepage] no focus style visible for UI Components and Visual Style Guide headers on homepage

### ISSUE THREAD

**Issue by carodew** [18F/web-design-standards#526]
**Steps to recreate**
1. on the homepage, tab to the headers "UI Components" and "Visual Style Guide" in the second section. 
2. note that focus disappears until you tab onto the red button "view the standards"

<img width="830" alt="screen shot 2015-09-15 at 11 08 31 am" src="https://cloud.githubusercontent.com/assets/10144074/9880637/70f2d03a-5b9a-11e5-8891-7b86b2f19ba0.png">

**Expected behavior**
The two headers should have a visible focus style. Re: https://pages.18f.gov/accessibility/keyboard/

**Tested on**
SauceLabs
Windows 10
IE 11

**Comment by angel** [comment]
No focus state on Chrome either.
